slash/general.py

```diff
@@ -32,6 +32,57 @@ async def github(self, interaction: Interaction):
             embed=embed, view=Link("https://github.com/codeman1o1/TIJK-Bot")
         )
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def avatar(
+        self,
+        interaction: Interaction,
+        user=SlashOption(
```

Comment:
`user` is not a Member type, but a string.
```suggestion
        user: nextcord.Member = SlashOption(
```

Reply:
Will change it soon